turbo.json

```diff
@@ -0,0 +1,34 @@
+{
+    "$schema": "https://turbo.build/schema.json",
+    "ui": "tui",
+    "tasks": {
+        "build": {
+            "dependsOn": ["^build"],
+            "outputs": ["dist/**", ".next/**"]
+        },
+        "lint": {
+            "dependsOn": ["//#eslint", "//#typecheck", "//#stylelint"]
+        },
+        "//#eslint": {
+            "outputs": ["node_modules/.cache/eslint"]
+        },
+        "eslint": {
+            "outputs": ["node_modules/.cache/eslint"]
+        },
+        "//#stylelint": {
+            "outputs": ["node_modules/.cache/stylelint"]
+        },
+        "stylelint": {
+            "outputs": ["node_modules/.cache/stylelint"]
+        },
+        "//#typecheck": {
+            "outputs": []
+        },
+        "typecheck": {
```

Comment:
Why no `outputs` ?

Add the following to your linting `tsconfig.json` file to get an output for `tsc`:

```
    "compilerOptions": {
        "incremental": true,
        "tsBuildInfoFile": "node_modules/.cache/tsbuildinfo.json",
    }
```

Reply:
Same comment for `//#typecheck`